library/src/main/java/com/devbrackets/android/exomedia/core/api/VideoViewApi.java

```diff
@@ -112,6 +112,10 @@
 
     void setTrack(ExoMedia.RendererType type, int trackIndex);
 
+    void setRendererTrackGroupIndex(ExoMedia.RendererType trackType, int trackGroupIndex);
```

Comment:
void setTrack(...) is causing the crash with "index out of bounds" exception when trying to select the first subtitle track (which is present in the stream):
`setTrack(ExoMedia.RendererType.CLOSED_CAPTION, 0);
When looking at the  "setTrack" implementation I have noticed that it might be used to select the different bandwidth tracks for video/audio.
Shall I change "setTrack" to make it work with the non-adaptive tracks instead?